docs/src/pages/[platform]/connected-components/storage/storage-browser/examples/ComposedDeleteView.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+import { createStorageBrowser } from '@aws-amplify/ui-react-storage/browser';
+import { mockConfig } from './mockConfig'; // IGNORE
+
+const { StorageBrowser, useView } = createStorageBrowser({
+  //...
+  config: mockConfig, // IGNORE
+});
```

Comment:
Think we should be good to provide the setup code once and omit this piece in the composed examples